src/main/kotlin/com/nikitanazarov/coroutinestacks/ui/CoroutineStacksPanel.kt

```diff
@@ -43,11 +35,14 @@ import javax.swing.JLabel
 class CoroutineStacksPanel(private val project: Project) : JBPanelWithEmptyText() {
     companion object {
         val dispatcherSelectionMenuSize = Dimension(200, 25)
+        const val maximumZoomLevel = 1f
```

Comment:
warning: Const property name 'maximumZoomLevel' should not contain lowercase letters